src/functions.js

```diff
@@ -159,6 +159,74 @@ defineFunction("\\color", {
     };
 });
 
+defineFunction("\\mathord", {
+    numArgs: 1,
+}, function(context, args) {
+   var body = args[0];
+
+   // Argument Parsing will give us an ordgroup when the arguments are
+   // enclosed by brackets.  We need to unwrap these if that's the case
+   // since TeX doesn't wrap {Args} in ord atoms when parsing for
+   // function arguments.
+   if (body.type === "ordgroup") {
+       // We already have an ordgroup, do nothing
+       return {
+           type: "mathord",
+           value: body.value,
+       }
+   } else {
+       return {
+           type: "mathord",
+           value: [body],
+       }
+   }
+});
+defineFunction("\\mathop", {
+    numArgs: 1,
+}, function(context, args) {
+   var body = args[0];
+
+   // Argument Parsing will give us an ordgroup when the arguments are
+   // enclosed by brackets.  We need to unwrap these if that's the case
+   // since TeX doesn't wrap {Args} in ord atoms when parsing for
+   // function arguments.
+   if (body.type === "ordgroup") {
+       // We already have an ordgroup, do nothing
+       return {
+           type: "op",
+           value: body.value,
+       }
+   } else {
+       return {
+           type: "op",
+           value: [body],
+       }
+   }
+});
+
+defineFunction("\\mathrel", {
+    numArgs: 1,
+}, function(context, args) {
+   var body = args[0];
+
+   // Argument Parsing will give us an ordgroup when the arguments are
+   // enclosed by brackets.  We need to unwrap these if that's the case
+   // since TeX doesn't wrap {Args} in ord atoms when parsing for
+   // function arguments.
+   if (body.type === "ordgroup") {
+       // We already have an ordgroup, do nothing
+       return {
+           type: "rel",
+           value: body.value,
+       }
+   } else {
+       return {
+           type: "rel",
+           value: [body],
+       }
+   }
```

Comment:
Oh good point I will do that.  I spent quite a bit of time fiddling around trying to figure out why atoms are wrapped twice (parseFunctions was culprit if I remember) and went through a few iterations to figure out what to put here.